src/contract.rs

```diff
@@ -2,21 +2,19 @@
 //! for sending transactions to contracts as well as querying current contract
 //! state.
 
-mod call;
 mod deploy;
-mod send;
+mod method;
 
 use crate::errors::{DeployError, LinkError};
 use crate::truffle::{Abi, Artifact, Bytecode};
-use ethabi::{Function, Result as AbiResult};
+use ethabi::{Result as AbiResult};
```

Comment:
This is what your linter is failing!

```suggestion
use ethabi::Result as AbiResult;
```